### PR TITLE
Add new film Rowing Woman

### PR DESCRIPTION
**Pull request by IvanTurchyn**

### CODE REVIEW

**Comment by IvanTurchyn** [comment]
Check my code :)

File: index.html [hunk 1]
```diff
@@ -64,6 +64,24 @@ <h1 class="title">Spider Man</h1>
 
     </section>
 
+        <section class="animate__animated animate__backInLeft top-gun">
```
Comment (lukasz2000) [inline index.html]: @IvanTurchyn all existing cards have same animate_backInLeft kind of animation - so maybe you will be able to change that to something else? 

To do that you need to go to https://animate.style website and pick different animate class than you have right now,ok? :)

File: index.html [hunk 2]
```diff
@@ -64,6 +64,24 @@ <h1 class="title">Spider Man</h1>
...
+        <div class="container">
+            <h1 class="title">Rowing Woman</h1>
+            <p class="description">Sara po burzliwej awanturze z chłopakiem traci dom i środki do życia. Zdesperowana kradnie samochód, który staje się jej jedynym schronieniem. Odkrywając jego zawartość, zakochuje się we właścicielu pojazdu i postanawia go odnaleźć. Wyrusza w podróż przez amerykańskie pustkowia, która pozwoli jej przewartościować życiowe pragnienia i priorytety.</p>
+            <img class="vod-logo" alt="Rowing-woman"
```
Comment (lukasz2000) [inline index.html]: You can also create images directory in your project and save the images instead of pull the external resource, ok?

**Comment by IvanTurchyn** [comment]
Check my code again :)

**Pull request closed without merging**